insights/parsers/openshift_get_with_config.py

```diff
@@ -14,44 +14,93 @@
+        admin                                                                 /admin                                                                                                                                                 openshift-infra/template-instance-controller
+        admin-0                                                               /admin                                                                                                                                                 kube-service-catalog/default
+        admin-1                                                               /admin                                                                                                                                                 openshift-ansible-service-broker/asb
+        asb-access                                                            /asb-access                                                                                                                                            openshift-ansible-service-broker/asb-client
+        asb-auth                                                              /asb-auth                                                                                                                                              openshift-ansible-service-broker/asb
+        auth-delegator-openshift-template-service-broker                      /system:auth-delegator                                                                                                                                 openshift-template-service-broker/apiserver
+        basic-users                                                           /basic-user                                                                                             system:authenticated
+        cluster-admin                                                         /cluster-admin                                                                                          system:masters
+        cluster-admin-0                                                       /cluster-admin                                                                                                                                         insights-scan/insights-scan
+
+    Examples:
+        >>> type(oc_get_clusterrolebinding_with_config)
+        <class 'insights.parsers.openshift_get_with_config.OcGetClusterRoleBindingWithConfig'>
+        >>> oc_get_clusterrolebinding_with_config.rolebinding["admin"]
+        '/admin'
+    """
 
     def parse_content(self, content):
-        self.rolebinding = {}
         if len(content) < 2 or "NAME" not in content[0]:
-            raise ParseException("{0}: invalid content".format(self.__class__.__name__))
+            raise ParseException("invalid content" if content else 'empty file')
+
+        self.data = parse_fixed_table(content)
```

Comment:
Either this `self.data` or the `self.rolebinding` is not necessary, you can just keep one of them.
`data=parser_fixed_table(content)`